src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -55,13 +57,17 @@ protected function configure()
   <info>php %command.full_name% framework</info>
   <info>php %command.full_name% FrameworkBundle</info>
 
+The <info>--format</info> option specifies the format of the configuration,
+this is either <comment>yaml</comment> or <comment>json</comment>.
```

Comment:
```suggestion
The <info>--format</info> option specifies the output format of the configuration,
this is either <comment>yaml</comment> or <comment>json</comment>.
When the option is not provided, <comment>yaml</comment> is used when the <comment>symfony/yaml</comment> component is available,
otherwise the <comment>json</comment> will be used.
```

maybe use the same dynamic `getAvailableFormatOptions` approach to generate available format options?

Reply:
I now use `getAvailableFormatOptions`, but don"t know a really adapt sentence.

> When the option is not provided, <comment>yaml</comment> is used when the <comment>symfony/yaml</comment> component is available,

Not sure if this detail is needed in help